src/viewport/viewport.model.ts

```diff
@@ -9,7 +9,8 @@ export enum ViewportSizeType {
 	medium = 2,
 	large = 3,
 	xlarge = 4,
-	xxlarge = 5,
+	xxlarge1 = 5,
+	xxlarge2 = 6
```

Comment:
why not `xxxlarge` ?

Reply:
having a numeric it might looks ugly at first, but in my opinion, adding `xxxlarge` is less scalable if we add more `xxxxxl` its hard to read and easy to misuse, also having numerics its quite often used https://getbootstrap.com/docs/4.0/content/typography/#headings, even in themes for darker colors are used

i suggest is:
- `xlarge`, `xlarge1` `xlarge2`... _first one no numeric + no xxl_
- `xlarge1`, `xlarge2` `xlarge3`... _starts with numeric + no xxl_
- `xlarge`, `xxlarge` `xxlarge1`... **(no breaking)** first one no numeric_
- `xlarge`, `xxlarge1` `xxlarge2`... _xl + xxl with first as numeric_

i would go for nonbreaking for now, if we find it better to be changed any of the other options we break when having a breaking change

Reply:
@stephenlautier @kurtaqui I went with the nonbreaking suggestion